javascript/javascript1/week3/Item-array-removal.js

```diff
@@ -0,0 +1,16 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+
+const nameToRemove = "Ahmad";
+
+names.splice(1,1);
```

Comment:
try to find value in the array with for loop

Reply:
//Thankyou @Ida i have updated.
const nameToRemove = "Ahmad";
//names.splice(1,1);
for (i=0; i<names.length; i++){
  if(names[i] === nameToRemove ){
    names.splice(i,1);
    break;
  }

}